<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZarzadzanieWycenami</class>
 <widget class="QWidget" name="ZarzadzanieWycenami">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTableView" name="tablica"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="ID_Label">
         <property name="text">
          <string>Id Wyceny</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ID_Line"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Cena_Label">
         <property name="text">
          <string>Cena</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cena_Line"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="Model_Label">
         <property name="text">
          <string>Model</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Model_Box"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="Usluga_Label">
         <property name="text">
          <string>Usluga</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Usluga_Box"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Wroc_Button">
       <property name="text">
        <string>Wróć</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Edytuj_Button">
       <property name="text">
        <string>Edytuj Cenę</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Usun_Button">
       <property name="text">
        <string>Usuń</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Dodaj_Button">
       <property name="text">
        <string>Dodaj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
